bloom-housing/bloom
# refactor: store email from address on jurisdiction entity

# Pull Request Template

## Issue Overview

This PR addresses #2383 


## Description

Moves the "email from address" from an environment variable to a string on the jurisdiction entity. The new strings are in the issue.

## How Can This Be Tested/Reviewed?

Locally switch your jurisdictions and perform an action that should send an e-mail to you (like creating a public account) and ensure the right string shows.

<img width="335" alt="Screen Shot 2022-02-08 at 5 55 04 PM" src="https://user-images.githubusercontent.com/65367387/153101750-6e95bb98-43b2-4f4d-aedb-6536edcc9b0f.png">
<img width="367" alt="Screen Shot 2022-02-08 at 5 55 17 PM" src="https://user-images.githubusercontent.com/65367387/153101752-372947a4-682a-4d47-ab16-1491cbe1256b.png">


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
